browser/ui/webui/brave_web_ui_controller_factory.cc

```diff
@@ -246,6 +257,10 @@ WebUIFactoryFunction GetWebUIFactoryFunction(WebUI* web_ui,
 #endif  // BUILDFLAG(IS_ANDROID)
 #if BUILDFLAG(ENABLE_TOR)
       url.host_piece() == kTorInternalsHost ||
+#endif
+#if BUILDFLAG(ENABLE_BRAVE_PLAYER)
+      (base::FeatureList::IsEnabled(brave_player::features::kBravePlayer) &&
+       url.host_piece() == kBravePlayerHost) ||
```

Comment:
Q) don't need to check private, tor profile or guest?

Reply:
Well, I don't find reason to disable this web ui/feature on the profiles unless adblock is disabled on the profiles.